### PR TITLE
mail_crypt_plugin: Mention mail_crypt_acl plugin

### PR DESCRIPTION
**Pull request by cmouse**

### CODE REVIEW

File: source/configuration_manual/mail_crypt_plugin.rst [hunk 1]
```diff
@@ -331,7 +331,8 @@ global key can be provided with several different scopes:
 
 With folder keys, key sharing can be done to single user, or multiple users.
 When key is shared to single user, and the user has public key available, the
-folder key is encrypted to recipient's public key.
+folder key is encrypted to recipient's public key. This requires `mail_crypt_acl` plugin,
+which will enable accessing encrypted shared folders.
```
Comment (fleischie) [inline source/configuration_manual/mail_crypt_plugin.rst:335]: Three comments:
1. I would appreciate it, if there were more articles: "[...] key sharing can be done to a single user [...]" and "When _a_ key is shared [...], has _a_ public key [...] to _the_ recipient's public key."; "This requires _the_ `mail_crypt_acl."
2. I have not found a reference to the "mail_crypt_acl" plugin anywhere, only to "mail_crypt_acl_require_secure_key_sharing".
3. The line exceeds 80 chars.

**Review by fleischie — CHANGES_REQUESTED** [review]

File: source/configuration_manual/mail_crypt_plugin.rst (hunk 1 above)
Comment (cmouse) [reply source/configuration_manual/mail_crypt_plugin.rst:335]: yeah, i'm kinda adding the reference here since we forgot to document it.

Comment (cmouse) [reply source/configuration_manual/mail_crypt_plugin.rst:335]: also the 80 character limit is soft limit, it does not need to be stricly enforced.

File: source/configuration_manual/mail_crypt_plugin.rst [hunk 2]
```diff
@@ -329,9 +331,10 @@ global key can be provided with several different scopes:
 * Global scope: key is configured in ``dovecot.conf`` file
 * Per-user(group) scope: key is configured in userdb file
 
-With folder keys, key sharing can be done to single user, or multiple users.
-When key is shared to single user, and the user has public key available, the
-folder key is encrypted to recipient's public key.
+With folder keys, key sharing can be done to single user, or to multiple users.
+When a key is shared to a single user, and the user has a public key available, the
+folder key is encrypted using recipient's public key. This requires the
+`mail_crypt_acl` plugin, which will enable accessing the encrypted shared folders.
```
Comment (sirainen) [inline source/configuration_manual/mail_crypt_plugin.rst:337]: double-`` right?

File: source/configuration_manual/mail_crypt_plugin.rst [hunk 3]
```diff
@@ -320,6 +320,8 @@ encrypt new mail, use ``mail_crypt_save_version=0``:
     mail_crypt_global_private_key = <server.key
   }
 
+.. _mail_crypt_acl_plugin:
```
Comment (sirainen) [inline source/configuration_manual/mail_crypt_plugin.rst:323]: Should this link be used somewhere also?

**Review by sirainen — CHANGES_REQUESTED** [review]

File: source/configuration_manual/mail_crypt_plugin.rst (hunk 3 above)
Comment (cmouse) [reply source/configuration_manual/mail_crypt_plugin.rst:323]: it is, line 14.

Comment (sirainen) [reply source/configuration_manual/mail_crypt_plugin.rst:323]: Oh, is that how it works?

**Review by sirainen — APPROVED** [review]

**Pull request merged**